linux/system-config/Makefile

```diff
@@ -7,5 +7,3 @@ install:
 	install -m 0755 create-snapshot destroy-snapshot $(DESTDIR)/usr/lib/qubes
 	install -m 0644 -D tmpfiles-qubes.conf $(DESTDIR)/usr/lib/tmpfiles.d/qubes.conf
 	install -d $(DESTDIR)/etc/logrotate.d
```

Comment:
```suggestion
```

Reply:
Why shouldn't `logrotate.d` be installed? It does contain other configs. Sorry if I miss a point here.

Reply:
I thought you had forgotten to push this patch hunk, because the commit message already said "Remove [...] empty parent directory".

> Why shouldn't `logrotate.d` be installed? It does contain other configs.

Those all come from other packages though. They'll take care of creating the directory themselves.

> Sorry if I miss a point here.

Admittedly it's kind of pointless OCD on my part :)

Reply:
> Those all come from other packages though. They'll take care of creating the directory themselves.

Ah I see, thanks for the explanation.